samples/sampler/stories/default/PageViews.js

```diff
@@ -17,7 +17,7 @@ export default {
 };
 
 export const _PageViews = (args) => (
-	<PageViews fullContents={args['fullContents']} pageIndicatorType={args['pageIndicatorType']}>
+	<PageViews fullContents={args['fullContents']} pageIndicatorPosition={args['pageIndicatorPosition']} pageIndicatorType={args['pageIndicatorType']}>
```

Comment:
I think we need to add `select` control for `pageIndicatorPosition`.

Reply:
It's fixed